src/app/draft/page.tsx

```
@@ -15,22 +15,37 @@ import React, { useState, useEffect } from 'react'
 
+type SimpleTab = BaseTab & {
+  component: React.FC
+  marketType?: never
+}
+
+type MarketTypeTab = BaseTab & {
+  component: React.FC<DraftMarketsQueueProps>
+  marketType: DraftMarketType
+}
+
+export type Tab = SimpleTab | MarketTypeTab
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Tab'. Consider to delete it or rename it.

'Tab' is defined here:


(lint/suspicious/noRedeclare)